.openpublishing.redirection.json

```diff
@@ -146,7 +146,7 @@
       "redirect_url": "/dotnet/azure/tools"
     },
     {
-      "source_path": "docs/azure/sdk/dotnet-tutorials.md",
+      "source_path": "docs/azure/dotnet-tutorials.md",
```

Comment:
I know @CamSoper recently removed the SDK directory. So do we need two redirects here instead?

Reply:
I talked to him offline. I was going to add an additional entry but he said the change on this line on the previous PR was an accident.